client/blocks/signup-form/social.jsx

```diff
@@ -102,25 +106,6 @@ class SocialSignupForm extends Component {
 						responseHandler={ this.handleAppleResponse }
 						onClick={ () => this.trackSocialLogin( 'apple' ) }
 					/>
-
-					<p className="signup-form__social-buttons-tos">
-						{ this.props.translate(
-							"If you continue with Google or Apple and don't already have a WordPress.com account, you" +
-								' are creating an account and you agree to our' +
-								' {{a}}Terms of Service{{/a}}.',
-							{
-								components: {
-									a: (
-										<a
-											href={ localizeUrl( 'https://wordpress.com/tos/' ) }
```

Comment:
Need to remove [the `localizeUrl` import](https://github.com/Automattic/wp-calypso/pull/36295/files#diff-0b8315ad95814790e533d7ecb7048b7aL18) now that it's not being used.